saleor/graphql/account/mutations/deprecated_account.py

```diff
@@ -0,0 +1,175 @@
+import graphene
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+
+from ....account import events as account_events, models, utils
+from ....checkout import AddressType
+from ...account.enums import AddressTypeEnum
+from ...account.types import Address, AddressInput, User
+from ...core.mutations import BaseMutation, ModelMutation
+from .base import BaseCustomerCreate, UserAddressInput, send_user_password_reset_email
+
+
+class CustomerRegisterInput(graphene.InputObjectType):
+    email = graphene.String(
+        description="The unique email address of the user.", required=True
+    )
+    password = graphene.String(description="Password", required=True)
+
+
+class CustomerRegister(ModelMutation):
+    class Arguments:
+        input = CustomerRegisterInput(
+            description="Fields required to create a user.", required=True
+        )
+
+    class Meta:
+        description = "DEPRECATED: Use AccountRegister instead. Register a new user."
+        exclude = ["password"]
+        model = models.User
+
+    @classmethod
+    def save(cls, info, user, cleaned_input):
+        password = cleaned_input["password"]
+        user.set_password(password)
+        user.save()
```

Comment:
```suggestion
        user.save(update_fields=["password"])
```

Reply:
We can't use update_fields for an object without primary key.